### PR TITLE
Closes #2 landing page

### LINKED ISSUE #2: Create a minimalistic landing page
Acceptance criteria:
* Landing page as `index.html`
* ~~The frontend should be a default React project~~ I was wrong, for now, it should be a static HTML page
* A design matching the screenshot provided below centered in the screen
* Font [Rubik](https://fonts.google.com/specimen/Rubik) is used

image

### PR DESCRIPTION
**Pull request by Vetzio**
These changes/additions address issue #2

### CODE REVIEW

**Comment by Vetzio** [comment]
@nizarmah I'm not sure if I need the <br> on line 23. Is there a break between the 2nd and 3rd lines of text in the example?

**Comment by nizarmah** [comment]
@Vetzio don't worry about matching it exactly. 

Just a general similarity is good enough. That's the point of the exercise.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <style>
+        .center-text {
+            text-align: center;
+        }
+        @font-face {
+            font-family: 'RubikFont';
+            src: url(https://github.com/MuleDerby/veles/blob/Issue-2-LandingPage/Fonts/Rubik-Regular.ttf);
```
Comment (nizarmah) [inline index.html:13]: You should use a relative path, for example `./Fonts/Rubik-Regular.ttf`

**Review by nizarmah — COMMENTED** [review]
Awesome work

Just fix the path please

File: index.html (hunk 1 above)
Comment (Vetzio) [reply index.html:13]: Ah! That makes sense. Working on it now 😃

**Review by nizarmah — APPROVED** [review]
Nice!!

**Comment by nizarmah** [comment]
Small note, PR titles should be informative of what the PR does, briefly.

For example: "Add basic landing page".

Just a note for the next PR 👍🏼

**Pull request merged**